test/Microsoft.AspNetCore.Server.KestrelTests/TestHelpers/MockLibuv.cs

```diff
@@ -76,6 +76,7 @@ unsafe public MockLibuv()
             _uv_strerror = errno => IntPtr.Zero;
             _uv_read_start = UvReadStart;
             _uv_read_stop = handle => 0;
+            _uv_unsafe_async_send = handle => 0;
```

Comment:
`=> _onPost(_postHandle.InternalGetHandle());` will probably work but it would be nice to know why this is getting called in the first place.